esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -2213,3 +2213,22 @@ extern TPWRAPPER_API void initContainerRoxieTargets(MapStringToMyClass<ISmartSoc
     }
 }
 
+extern TPWRAPPER_API unsigned getContainerThorClusterNames(StringArray& targetNames, StringArray& queueNames)
+{
+    Owned<IStringIterator> targets = getContainerTargetClusters("thor", nullptr);
+    ForEach(*targets)
+    {
+        SCMStringBuffer target;
+        targets->str(target);
+        targetNames.append(target.str());
+        StringBuffer queueName(target.str());
```

Comment:
Very minor optimization, but you do have access to target.s and don't have to create a second StringBuffer queueName.

```
SCMStringBuffer target;
targets->str(target);
targetNames.append(target.str());
queueNames.append(target.s.append(THOR_QUEUE_EXT));
```